build.sbt

```diff
@@ -69,13 +69,35 @@ lazy val `lang-tests-js` = project
     testFrameworks += new TestFramework("utest.runner.Framework")
   )
 
-lazy val node = project.dependsOn(`lang-jvm`, `lang-testkit` % "test;test->test")
+lazy val node = project.dependsOn(`lang-jvm`)
 
-lazy val `grpc-server`    = project.dependsOn(node % "compile;test->test;runtime->provided")
-lazy val `ride-runner`    = project.dependsOn(node % "compile;test->test", `grpc-server`)
-lazy val `node-it`        = project.dependsOn(node % "compile;test->test", `lang-testkit`, `repl-jvm`, `grpc-server`)
-lazy val `node-generator` = project.dependsOn(node % "compile->test")
-lazy val benchmark        = project.dependsOn(node % "compile;test->test")
+lazy val `node-testkit` = project
+  .in(file("node/testkit"))
+  .dependsOn(`lang-jvm`, `lang-testkit` % "compile->compile", `node` % "compile->test")
+  .settings(
+    libraryDependencies ++=
+      Dependencies.test.map(_.withConfigurations(Some("compile"))) ++ Dependencies.qaseReportDeps ++ Dependencies.logDeps ++ Seq(
+        "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5"
+      )
+  )
+lazy val `node-tests` = project
+  .in(file("node/tests"))
+  .dependsOn(`lang-jvm`, `node`, `lang-testkit` % "test;test->test", `node-testkit`)
+  .settings(libraryDependencies ++= Dependencies.node.value)
+
+lazy val `grpc-server` =
+  project.dependsOn(node % "compile;test->test;runtime->provided", `node-testkit`, `node-tests` % "compile->test")
+lazy val `ride-runner` = project.dependsOn(node % "compile;test->test", `grpc-server`, `node-tests` % "compile->test")
+lazy val `node-it` = project.dependsOn(
+  node % "compile;test->test",
+  `lang-testkit`,
+  `node-testkit` % "compile->test",
+  `repl-jvm`,
+  `grpc-server`,
+  `node-tests` % "compile->test"
```

Comment:
`node-it` has only tests. Also, this test code depends on tests of `node-tests` (`EthTxGenerator`), which depends both on `lang-testkit` and `node-testkit`. I think:
```suggestion
  `repl-jvm`,
  `grpc-server`,
  `node-tests` % "test->test"
```
looks more natural.

**BUT, probably, I'd better** to have `EthTxGenerator` in `node-testkit`. @phearnot , what do you think?